<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="menuLable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>750</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>750</width>
     <height>750</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>750</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bernard MT</family>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>240</y>
     <width>240</width>
     <height>110</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>110</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>110</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>48</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>430</y>
     <width>240</width>
     <height>110</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>110</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>48</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="themeLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>591</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>48</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tower Defence</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
